util/linuxfw/fake.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build linux
```

Comment:
Maybe we could add a 'test' build tag if we want to avoid including these fakes into the compiled tailscale? Not a big deal if we do I guess it's not that big